DEPS

```diff
@@ -18,6 +18,7 @@ deps = {
   "third_party/externals/spirv-cross"     : "https://chromium.googlesource.com/external/github.com/KhronosGroup/SPIRV-Cross@bdbef7b1f3982fe99a62d076043036abe6dd6d80",
   #"third_party/externals/v8"              : "https://chromium.googlesource.com/v8/v8.git@5f1ae66d5634e43563b2d25ea652dfb94c31a3b4",
   "third_party/externals/zlib"            : "https://chromium.googlesource.com/chromium/src/third_party/zlib@eaf99a4e2009b0e5759e6070ad1760ac1dd75461",
+  "third_party/externals/brotli"          : "https://skia.googlesource.com/external/github.com/google/brotli.git@e61745a6b7add50d380cfd7d3883dd6c62fc2c71",
```

Comment:
You'll have to make a PR to update `rust-skia` to point to this. Although I'm wondering why do we need to fork `skia`? can't we base it on a `rust-skia` release that has woff2 and brotli?

Reply:
>You'll have to make a PR to update rust-skia to point to this

Sure, I'll do that. A small adjustment needs to be done in `rust-skia` as well, I'll open a PR that both does that adjustment and points to skia with brotli

>Although I'm wondering why do we need to fork skia? can't we base it on a rust-skia release that has woff2 and brotli?

We have to fork skia because:
- We disable hinting for text: https://github.com/romanzes/skia/commit/24c5eadc2c74ef4edc290f9e08b5b987c01db730
- We adjust text gamma: https://github.com/romanzes/skia/commit/1c086aace5c18fe051d09dd85488a496a2c25a99
- `rust-skia`'s fork doesn't have brotli in the dependencies

Reply:
Cool, it’d be nice to expose bindings to do the hinting and gamma rather than fork skia if possible. That way we may be able to merge them upstream.

The gamma defines we should be able to pass from rust-skia.

Reply:
>it’d be nice to expose bindings to do the hinting

I think we'd have to merge a PR to `google/skia` that allows you to configure fonts used in the text layout, which would require a serious refactoring. I reported it to them, can't find the thread now, hopefully one day they'll get to it.

Reply:
Ah, that's a pity - I guess we fork for now